packages/controller-utils/package.json

```diff
@@ -31,7 +31,9 @@
   "dependencies": {
     "@metamask/utils": "^5.0.2",
     "@spruceid/siwe-parser": "1.1.3",
+    "babel-runtime": "^6.26.0",
```

Comment:
Good point. I included it because we have a constraint requiring it. Agreed that it's not ideal.

I have instead opted to avoid adding either dependency by inlining a more lightweight version of the EthQuery type: https://github.com/MetaMask/core/pull/1266/commits/601a61abe7a9bdf72441685dbeae21690d61c420